source/shared/smart-contracts/tutorials/piggy-bank/testing.rst

```diff
@@ -80,24 +80,27 @@ contract.
 Preparation
 ===========
 
-Before we start, make sure to have the necessary tooling for building Rust
+Before you start, make sure to have the necessary tooling to build Rust
 contracts.
-The guide :ref:`setup-tools` will show you how to do this.
-Also, make sure to have a text editor setup for writing Rust.
+The guide :ref:`setup-tools` shows you how to do this.
+Also, make sure to have a text editor setup to write Rust.
 
-Since we are going to extend the smart contract code written in the previous
-part, either follow the previous part or copy the resulting code from there.
+Additionally, to run the tests you need to:
 
-.. todo::
+- set up a local testnet node using your preferred platform: :ref:`Windows<windows-node>`, :ref:`MacOS<macos-node>`, :ref:`Ubuntu<ubuntu-node>`, or :ref:`Docker/Linux<docker-node>`
+- :ref:`create an account for testnet<create-account-desktop>`. The account will need some CCD to run tests.
```

Comment:
Do we have instructions for requesting GTU drop? If so, can we link them.

Reply:
I'm not sure I've seen any instructions for requesting CCDs but it would be good to have.